msteams-platform/m365-apps/cards-loop-component.md

```diff
@@ -0,0 +1,100 @@
+
+Adaptive Card-based Loop components enable you to build collaborative experiences within your Teams [message extensions](../messaging-extensions/what-are-messaging-extensions.md) that work across Microsoft 365. Adaptive Card-based Loop components have actionable content that enables users to make live updates without having to switch context between Microsoft 365 apps, such as Teams and Outlook.
+
+[Loop components](https://support.microsoft.com/office/first-things-to-know-about-loop-components-in-microsoft-teams-ee2a584b-5785-4dd6-8a2d-956131a29c81) were first released in Teams followed by other Microsoft 365 apps such as Outlook, Whiteboard, and Loop app. Loop components allow users to collaborate and share live content with others in the chat, email, meeting, Whiteboard or Loop app. Because Loop components stay in sync across different Microsoft 365 apps, they enable users to coauthor content and make real-time updates on their content. For more information, see [Overview of Loop components in the Microsoft 365 ecosystem](/microsoft-365/loop/loop-components-teams).
+
+Following is an example of an Adaptive Card-based Loop component:
+
+:::image type="content" source="~/assets/images/adaptive-cards/adaptive-card-loop.png" alt-text="Example of an Adaptive Card loop component.":::
+
+This article provides an overview of how to build and test your Adaptive Card-based Loop components.
+
+## Prerequisites
+
+Before you build an Adaptive Card-based Loop component, ensure that you meet the following prerequisites:
+
+1. [Build a Message Extension with a Search command](../messaging-extensions/what-are-messaging-extensions.md).
+
+1. Add [link unfurling](../messaging-extensions/how-to/link-unfurling.md) support to the Message Extension.
+
+1. Use [Universal Actions for Adaptive Cards](../task-modules-and-cards/cards/Universal-actions-for-adaptive-cards/Work-with-Universal-Actions-for-Adaptive-Cards.md).
+
+1. [Extend your Teams message across Microsoft 365](extend-m365-teams-message-extension.md).
+
+## Build Adaptive Card-based Loop component
+
+After you've met all the requirements, upgrade the Adaptive Card into a Loop component using the following steps:
+
+1. Ensure that the Adaptive Card adheres to the [design guidelines](design-loop-components.md) to build an actionable and coherent Adaptive Card-based experience for your end users.
+1. To enable Loop component, add the URL that uniquely identifies the card in the [metadata.webUrl](https://adaptivecards.io/explorer/Metadata.html) property in the [Adaptive Card schema](https://adaptivecards.io/explorer/). The `metadata.webUrl` property supports portability via the Copy button present in the Loop component header.
+
+### Example
+
+The following is a JSON example of an Adaptive Card-based Loop component with the `metadata` and `webUrl` properties:
+
+```json
+{
+  "type": "AdaptiveCard",
+  "version": "1.6",
+  "metadata": {
+    "webUrl": "https://msdn.microsoft.com/microsoft-teams/tabs"
```

Comment:
Can we use an approved fictitious domain here, like: `https://contoso.com/tab`

Reply:
updated.